tests/services/test_service_users.py

```diff
@@ -136,3 +137,93 @@ def test_read_self(user_service, users, username):
     assert res["email"] == user.email
     # Avatar can also be seen.
     user_service.read_avatar(user.identity, user.id)
+
+
+def test_search_permissions(app, db, user_service, user_moderator, user_res):
+    """Test service search for permissions."""
+    # User can search for himself
+    search = user_service.search(
+        user_res.identity, q=f"username:{user_res._user.username}"
+    )
+    assert search.total > 0
+    # TODO user should see  moderation fields such as 'verified_at'?
```

Comment:
question: if an user searches for itself, should the user see fields such as `verified_at` or `blocked_at`?